publication-commons/src/main/java/no/unit/nva/publication/service/impl/ResourceService.java

```diff
@@ -110,7 +110,6 @@ public class ResourceService extends ServiceWithTransactions {
     private static final Logger logger = LoggerFactory.getLogger(ResourceService.class);
     private final String tableName;
     private final Clock clockForTimestamps;
-    private final Supplier<SortableIdentifier> identifierSupplier;
```

Comment:
Removing identifier supplier as it is not used everywhere and is tested only to make sure that we do not allow inserting publication with the same identifier twice. Testing it directly in ServiceWithTransaction instead and removing it from ResourceService.